src/admin/components/views/collections/Edit/Default.tsx

```diff
@@ -92,8 +94,8 @@ const DefaultEditView: React.FC<Props> = (props) => {
           >
             <div className={`${baseClass}__main`}>
               <Meta
-                title={`${isEditing ? 'Editing' : 'Creating'} - ${collection.labels.singular}`}
-                description={`${isEditing ? 'Editing' : 'Creating'} - ${collection.labels.singular}`}
+                title={`${isEditing ? t('editing') : t('creating')} - ${collection.labels.singular}`}
```

Comment:
The labels (`collection.labels.singular`) need to be parsed through the `getTranslation` function since they can be objects.